helm/tableland/values.yaml

```diff
@@ -0,0 +1,306 @@
+            "EthEndpoint": "wss://opt-mainnet.g.alchemy.com/v2/${VALIDATOR_ALCHEMY_OPTIMISM_MAINNET_API_KEY}",
+            "ContractAddress": "0xfad44BF5B843dE943a09D4f3E84949A11d3aa3e6"
+          },
+          "EventFeed": {
+            "ChainAPIBackoff": "15s",
+            "NewBlockPollFreq": "5s",
+            "MinBlockDepth": 0,
+            "PersistEvents": true
+          },
+          "EventProcessor": {
+            "BlockFailedExecutionBackoff": "10s",
+            "DedupExecutedTxns": true,
+            "WebhookURL": "https://discord.com/api/webhooks/${VALIDATOR_DISCORD_WEBHOOK_ID}/${VALIDATOR_DISCORD_WEBHOOK_TOKEN}"
+          },
+          "HashCalculationStep": 1800
+        },
+        {
+          "Name": "Filecoin Mainnet",
+          "ChainID": 314,
+          "Registry": {
+            "EthEndpoint": "https://node.glif.io/fvm-archive/lotus/rpc/v1",
+            "ContractAddress": "0x59EF8Bf2d6c102B4c42AEf9189e1a9F0ABfD652d",
+            "ProviderAuthToken": "${VALIDATOR_GLIF_FILECOIN_MAINNET_API_KEY}"
+          },
+          "EventFeed": {
+            "ChainAPIBackoff": "15s",
+            "NewBlockPollFreq": "15s",
+            "MinBlockDepth": 5,
+            "PersistEvents": true
+          },
+          "EventProcessor": {
+            "BlockFailedExecutionBackoff": "10s",
+            "DedupExecutedTxns": true,
+            "WebhookURL": "https://discord.com/api/webhooks/${VALIDATOR_DISCORD_WEBHOOK_ID}/${VALIDATOR_DISCORD_WEBHOOK_TOKEN}"
+          },
+          "HashCalculationStep": 60
+        }
+      ]
+    }
+
+
+extraEnvs: 
+  - name: VALIDATOR_ALCHEMY_ARBITRUM_MAINNET_API_KEY
+    value: secret-value
+  - name: VALIDATOR_ALCHEMY_ETHEREUM_MAINNET_API_KEY
+    value: secret-value
+  - name: VALIDATOR_ALCHEMY_POLYGON_MAINNET_API_KEY
+    value: secret-value
+  - name: VALIDATOR_ALCHEMY_OPTIMISM_MAINNET_API_KEY
+    value: secret-value
+  - name: VALIDATOR_GLIF_FILECOIN_MAINNET_API_KEY
+    value: secret-value
+  - name: VALIDATOR_QUICKNODE_ARBITRUM_NOVA_MAINNET_API_KEY
+    value: secret-value
+  - name: HTTP_RATE_LIMITER_API_KEY
+    value: very-secret
+  - name: METRICS_HUB_API_KEY
+    value: api-key
+   
+envFrom: []
+# - secretRef:
+#     name: env-secret
+# - configMapRef:
+#     name: config-map
+
+secretMounts: []
+#  - name: tls-certificates
+#    secretName: tls-certificates
+#    path: /.tableland//certs
+#    defaultMode: 0755
+
+rbac:
+  create: true
+  serviceAccountAnnotations: {}
+  serviceAccountName: ""
+  automountToken: true
+
+persistence:
+  enabled: true
+  annotations: {}
+
+extraVolumeMounts: []
+# - name: extras
+#   mountPath: /usr/share/extras
+#   readOnly: true
+
+extraContainers: []
+# - name: do-something
+#   image: busybox
+#   command: ['do', 'something']
+
+priorityClassName: ""
+enableServiceLinks: false
+
+# Http port of the application
+httpPort: 8080
+# Https port of the application
+httpsPort: 8443
+# Metrics port of the application
+metricsPort: 8888
```

Comment:
the [default validator metrics port is 9090](https://github.com/tablelandnetwork/go-tableland/pull/606/files#diff-bd4a53e670a2ca46ca2e6469440b98cee40f3540d0667db5acfaea6523dad609R47), if you change this to 9090, metrics can be accessed at `:9000/metrics`

if you change this, there's a change that needs to be made in the [readme](https://github.com/tablelandnetwork/go-tableland/pull/606/files#diff-5cf4da3bcb0434f460246bd0205f3e28be2f8f09e10d355065de3d55b0397761R44) too